### PR TITLE
Update README.md file in the azure-spring-boot-sample-active-directory-resource-server

### PR DESCRIPTION
**Pull request by ZhuXiaoBing-cn**
Update the webapi README.md file.

### CODE REVIEW

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/src/main/java/com/azure/spring/sample/aad/controller/HomeController.java [hunk 1]
```diff
@@ -16,14 +16,14 @@
     @GetMapping("/file")
     @ResponseBody
     @PreAuthorize("hasAuthority('SCOPE_File.read')")
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/src/main/java/com/azure/spring/sample/aad/controller/HomeController.java:18]: To keep the same style, please use `File.Read`.

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md [hunk 2]
```diff
@@ -3,109 +3,102 @@
...
-1. Sign in to the [Azure portal](https://portal.azure.com/) with either a work or school account or a personal Microsoft account.
-2. If your account is present in more than one Azure AD tenant, select your profile at the upper right, and then select **Switch directory**.
-3. Change your portal session to the Azure AD tenant you want to use.
-
-### Register the web API
-
-1. Go to the Microsoft identity platform for developers App registrations portal.
+### Configure web api
+1. Search for and select your tenant in **Azure Active Directory**.
+2. Under Manage In the same tenant, select **App registrations** -> **New registration**.![Protal manage](docs/image-protal-manage.png "Protal manage")
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md:18]: we may change 
"Under Manage In the same tenant" 
--->
"Under **Manage**"

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md [hunk 3]
```diff
@@ -3,109 +3,102 @@
...
-   - For **Who can consent**, ensure that the **Admins and users** option is selected.
-   - In the **Admin consent display name** box, enter **Access File.read as a user**.
-   - In the **Admin consent description** box, enter **Accesses the File.read web API as a user**.
-   - In the **User consent display name** box, enter **Access File.read as a user**.
-   - In the **User consent description** box, enter **Accesses the File.read web API as a user**.
-   - For **State**, keep **Enabled**.
-   - Select **Add scope**.
-   ![Scope Config](docs/image-scope-configurations.png "Scope Config")
+### Configure web app
+If you don't know web app at the moment, you can look at this [azure-spring-boot-sample-active-directory-webapp].
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md:28]: "If you don't know web app at the moment, you can look at this [azure-spring-boot-sample-active-directory-webapp]."
--->
"If you are not familiar with web app , you can refer this [azure-spring-boot-sample-active-directory-webapp]."

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md [hunk 4]
```diff
@@ -3,109 +3,102 @@
...
+At this point, I believe you have understood the web app, so let's continue!
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md:29]: > At this point, I believe you have understood the web app, so let's continue!
I think we can delete this sentence.

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md [hunk 5]
```diff
@@ -3,109 +3,102 @@
...
+
+azure:
+  activedirectory:
+    client-id: <client-id>
+    app-id-uri: <app-id-uri>
+```
+**NOTE**：Set the `server.port` in the application.yml of `webapp` and `webapi` to different value.
+
+### Add code
+Add code(Here code is just an example, let you understand the process.) to webapp's `ClientController.java`.
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md:63]: "Here code is just an example, let you understand the process."
-->
"The code here is just an example to show the process"

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md [hunk 6]
```diff
@@ -3,26 +3,19 @@
...
 This sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts only.
 
-1. The bearer token is obtained from the request header. 
+1. The bearer token is obtained from the request header.
 2. `JwtDecoder`  is used to parse the token into `Jwt`.
-3. Claims, headers etc in `Jwt` will be extracted, they will be wrapped in `AzureOAuth2AuthenticatedPrincipal` object.
-4. `AzureOAuth2AuthenticatedPrincipal` will eventually be set into SecurityContext.
-
+3. Verify `aud`, `iss`, `nbf`, `exp` in access token.
+4. After successful verification, claims, headers etc in `Jwt` will be extracted, they will be wrapped in `AzureOAuth2AuthenticatedPrincipal` object.
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md]: We've changed this class's name.

Comment (saragluna) [reply sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md]: How about change this sentence to `Extract information from JWT in xxx object after a successful verification` ?

Comment (saragluna) [reply sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md]: I've updated these directly. Resolve this conversation now.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**